app/keepers/keepers.go

```diff
@@ -403,12 +421,16 @@ func NewAppKeeper(
 
 	// Add transfer stack to IBC Router
 
-	// Create Interchain Accounts Stack
+	// Create Interchain Accounts Host Stack
 	var icaHostStack porttypes.IBCModule = icahost.NewIBCModule(appKeepers.ICAHostKeeper)
 
+	// Create Interchain Accounts Controller Stack
+	var icaControllerStack porttypes.IBCModule = icacontroller.NewIBCMiddleware(nil, appKeepers.ICAControllerKeeper)
```

Comment:
Because it contains implementation of IBC functionality of Controller module (e.g. channel handshake and sending of packets). This might be confusing because of the function name- it is usually called just "NewIBCModule()" for other modules. But the Controller can also act as a middleware if you provide an underlying application to it (if you want to execute some custom callbacks when IBC messages are handled by the Controller, e.g. OnChanOpenInit() etc.)

Reply:
That makes sense. Thanks!